### PR TITLE
Messaging GA release

### PR DESCRIPTION
**Pull request by spoorthipujariadobe**

### CODE REVIEW

File: release-notes/README.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## June 17, 2021
```
Comment (emdobrin) [inline release-notes/README.md:7]: I believe this was released on June 16th, @ravjain-adb can you confirm?

Comment (ravjain-adb) [reply release-notes/README.md:7]: 17 is good we did android on 16 and ios on 17.

Comment (emdobrin) [reply release-notes/README.md:7]: or we can split this in two parts one for iOS and one for Android ? I am ok this way too.

**Comment by emdobrin** [comment]
@spoorthipujari @ravjain-adb see my comment above, let's decide on one way and merge this in.

File: release-notes/README.md (hunk 1 above)
Comment (ravjain-adb) [reply release-notes/README.md:7]: Yup we can divide the release. 
@spoorthipujariadobe lets make android at 16 and ios at 17

**Review by emdobrin — APPROVED** [review]

**Pull request merged**